cfgrammar/src/lib/yacc/parser.rs

```diff
@@ -43,7 +43,8 @@ pub enum YaccParserErrorKind {
     DuplicateAvoidInsertDeclaration(Vec<Span>),
     /// Contains the spans of all duplicated implicit token declarations.
     DuplicateImplicitTokensDeclaration(Vec<Span>),
-    DuplicateExpectDeclaration,
+    /// Contains the spans of all duplicated expect declarations.
```

Comment:
`duplicated` -> `duplicate`.  been using the wrong tense here in a few places.

Reply:
Good point.